pkg/kubelet/apis/config/types.go

```diff
@@ -67,6 +67,12 @@ const (
 	// SingleNumaNodeTopologyManager Policy iis a mode in which kubelet only allows
```

Comment:
unrelated to this PR, but mentioning anyway:

```suggestion
	// SingleNumaNodeTopologyManager Policy is a mode in which kubelet only allows
```

Reply:
Done